### PR TITLE
Updated versioning image to match stencils

### PR DESCRIPTION
**Pull request by dvdstelt**
Existing architecture/scaling.md improved with better image and converted it to .png.

Changes in most files are simply updated references to new page. I noticed there was an existing (older) page already under /transports/ which contained [outdated and duplicate content](https://github.com/Particular/docs.particular.net/pull/4390#issuecomment-501596839).

### CODE REVIEW

**Comment by dvdstelt** [comment]
Maybe add a menu item somewhere?

**Comment by dvdstelt** [comment]
Maybe delete this page? It contains outdated information which is also split across multiple new documents.
Like https://docs.particular.net/transports/types and https://docs.particular.net/monitoring/metrics/

File: nservicebus/architecture/scaling.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ---
 title: Scaling with NServiceBus
 summary: NServiceBus provides several options to scale out a system
-reviewed: 2019-02-08
+tags:
+ - scalability
+ - routing
+redirects:
+ - nservicebus/architecture/scaling
```
Comment (danielmarbach) [inline nservicebus/architecture/scaling.md]: Shouldn't this be `transports/scale-out`?

Comment (dvdstelt) [reply nservicebus/architecture/scaling.md]: Good catch!

**Pull request merged**